shell/browser/api/electron_api_web_contents.cc

```diff
@@ -1373,8 +1373,10 @@ void WebContents::HandleNewRenderFrame(
   // Set the background color of RenderWidgetHostView.
   auto* web_preferences = WebContentsPreferences::From(web_contents());
   if (web_preferences) {
-    web_contents()->SetPageBaseBackgroundColor(
-        web_preferences->GetBackgroundColor());
+    absl::optional<SkColor> color = web_preferences->GetBackgroundColor();
+    web_contents()->SetPageBaseBackgroundColor(color);
+    web_contents()->GetRenderWidgetHostView()->SetBackgroundColor(
+        color.value_or(SK_ColorWHITE));
```

Comment:
Maybe we should create `WebContents::SetBackgroundColor(absl::optional<SkColor> color)` to reduce the chance of this being modified only in one place.